src/java/com/trade/delivery/SumOfPlanes.java

```diff
@@ -4,18 +4,21 @@
 
 public class SumOfPlanes {
     public static void main(String[] args) {
-        System.out.println("Your number: "+ sum());
+        System.out.println("Your number: " + sum());
     }
-    /* Creating a variable int sum. It calculates the sum of two numbers*/
+    //Creating a variable int sum. It calculates the sum of two numbers
     public static int sum () {
         Scanner scan = new Scanner(System.in);
         System.out.println("Enter the first number: ");
         int a = scan.nextInt();
         System.out.println("Enter the second number: ");
         int b = scan.nextInt();
-        int sum = a+b;
+        int sum = a + b;
         return sum;
     }
-
-
+    //Created a new method that takes two parameters at the input
+    public Integer sumOfNumbers (int firstNumbers, int secondNumbers){
+        int sum = firstNumbers + secondNumbers;
```

Comment:
Try to write more code. Make just return the sum of numbers without creating a new variable.